File: EventsApp.Server/EventsApp.API/Data/ValidationConstants.cs
namespace EventsApp.API.Data
{
    public static class ValidationConstants
    {
        public static class User
        {
            public const int FirstNameMaxLength = 50;

            public const int FirstNameMinimumLength = 2;

            public const int LastNameMaxLength = 50;

            public const int LastNameMinimumLength = 2;
        }

        public static class Event
        {
            public const int NameMaxLength = 50;

            public const int NameMinLength = 2;

            public const int SportMaxLength = 60;

            public const int SportMinLength = 2;

            public const int LocationMinLength = 2;
        }

        public static class Position
        {
            public const int NameMaxLength = 60;

            public const int NameMinLength = 2;
        }

        public static class Publication
        {
            public const int DescriptionMaxLength = 200;
        }
    }
}

File: EventsApp.Client/src/app/core/validation-constants.ts
export class User {
  static FIRST_NAME_MAX_LENGTH = 50;
  static FIRST_NAME_MIN_LENGTH = 2;
  static LAST_NAME_MAX_LENGTH = 50;
  static LAST_NAME_MIN_LENGTH = 2;
  static PASSWORD_MIN_LENGTH = 6;
  static MOBILE_MIN_LENGTH = 10;
  static MOBILE_MAX_LENGTH = 15;
  static FACEBOOK_URL_MIN_LENGTH = 25;
}

export class Event {
  static TITLE_MAX_LENGTH = 50;
  static TITLE_MIN_LENGTH = 2;
  static SPORT_MAX_LENGTH = 60;
  static SPORT_MIN_LENGTH = 2;
  static LOCATION_MIN_LENGTH = 2;
  static AVAILABLE_POSITIONS_MIN_LENGTH = 1;
}

export class Position {
  static TITLE_MAX_LENGTH = 60;
  static TITLE_MIN_LENGTH = 2;
}

export class Publication {
  static DESCRIPTION_MAX_LENGTH = 200;
}
